config/locales/en.yml

```diff
@@ -170,6 +186,29 @@ en:
       percentage_owned:
         input: What percentage share of their home does your client legally own?
         input_hint: Your client's name must be on the property deeds, lease or freehold.
+    vehicle_value:
+      caption: Yout clients finances
+      heading: Add details about your clients vehicle
```

Comment:
`client's` - apostrophe missing

Reply:
Actually I have no idea. We're currently inconsistent - I was about to say this way was right, and that we should ask James' opinion

Reply:
I've fixed up all the inconstistencies